examples/values-ez.yaml

```diff
@@ -114,6 +163,52 @@ scaffold:
         pullPolicy: IfNotPresent
         version: latest
 
+    logSigner:
+      image:
+        registry: quay.io
+        repository: securesign/trillian_log_signer
+        version: v1.2.2
+        pullPolicy: IfNotPresent
+    logServer:
+      image:
+        registry: quay.io
+        repository: securesign/trillian_log_server
+        version: v1.2.2
+        pullPolicy: IfNotPresent
+    mysql:
+      gcp:
+        scaffoldSQLProxy:
+          registry: quay.io
+          repository: securesign/cloudsqlproxy
+          version: v0.6.4
+      image:
+        registry: quay.io
+        repository: securesign/trillian-db
+        version: v1.5.2
+        pullPolicy: IfNotPresent
+      args: []
```

Comment:
Yes the changes made for the args is specific as the argument "--ignore-db-dir=lost+found" is now [deprecated](https://stackoverflow.com/questions/49945091/latest-mysql-not-coming-up-in-kubernetes), the securityContext:fsGroup: 0 is because I ran into some permission issues when trying to create directory's for MySQL.

And the liveness/readiness probe changes is actually a issue with the universal base image, that status command is used to check if the mysql instance is up and running, However the ubi doesnt have that cmd, so using the mysqladmin ping cmd should do the same thing, as far as I know

Reply:
ok!